requirements-minimal.txt

```diff
@@ -1,7 +1,5 @@
 cached-property
-# TODO: stop using our fork
-# https://github.com/celery/celery/pull/3831
-ckuehl-celery[redis]==4.0.2.post1
+celery[redis]==4.1.0
```

Comment:
do we need to pin here?

also need to update requirements.txt (or run make update-requirements) for this to actually do anything